alsoHeresTheInterviewQuestionIHad/alsoHeresTheInterviewQuestionIHad.js

```diff
@@ -0,0 +1,42 @@
+const fs = require("fs");
+const streetArray = fs.readFileSync("data.txt", "utf-8").split(",");
+
+let answer = "";
+
+let numberOfBuildings = streetArray.length;
+let tallestBuildingSoFar = 0;
+let tallestBuilding = Math.max(...streetArray);
+
+let bannerOneHeight = tallestBuilding;
+let bannerOneWidth = numberOfBuildings;
+let bannerOneArea = tallestBuilding * numberOfBuildings;
+let bannerTwoHeight = tallestBuilding;
+let bannerTwoWidth = numberOfBuildings;
+let bannerTwoArea = tallestBuilding * numberOfBuildings;
+
+let bestArea = bannerOneArea;
+
+function updateBanners(address){

```

Comment:
Address is fine I suppose, I personally prefer index if we're talking about positions in an array

Reply:
This is something I'll have to reflect upon.  In this context, I thought address was pretty acceptable as it's analogous to us  walking down a street in front of the buildings.  Indeed, that's literally how the problem is described?